HowToVenv/README.md

```diff
@@ -0,0 +1,110 @@
+## Main information docs: 
+https://docs.python.org/3/library/venv.html
+
+https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment
+
+
+## Why to use virtual environment?
+To isolate environment from the global environments and having same Python pip libraries as the project obtains is what venv is mainly used for. 
+Sometimes the project may require specific libraries to work as syntax of the libraries may change or they are depended on another older libraries.
+
+Below in the picture the first pip3 freeze shows the pip libraries in the virtualenvironment scope. 
+Next, when deactivating the virtual environment, the global pip libraries of the system is a looong list of different library versions. 
+
+![Virtualenv differences](virtualenv-diff.png)
+
+## Commands to run virtual environments
+Note: It's easiest to be in the main folder to do the commands -> /beginners-python-weather
+
+Note 2: first venv = the command & later venv = the environment name
+
+`py/python3 -m venv(command) venv(name)`
+
+### Windows
+
+**Command Prompt:**
+
+Create a venv
+```
+py -m venv venv
```

Comment:
Quite unusual to have python3 aliased as py?

Reply:
Windows magic it seems for pointing out the file path. Gives error with python/3 alias, but guess there are some workarounds for that. Tested it working locally